internal/events/subscription.go

```diff
@@ -214,6 +215,7 @@ func (s *subscription) createFilter(ctx context.Context, since *big.Int) error {
 		return errors.Errorf(errors.RPCCallReturnedError, "eth_newFilter", err)
 	}
 	s.catchupBlock = nil // we are not in catchup mode now
+	s.info.Synchronized = true
```

Comment:
I think we should wait to flick this to true until we've done our first `eth_getFilterLogs` and switch to `eth_getFilterChanges`:

https://github.com/hyperledger/firefly-ethconnect/blob/e4bd74893218ddd38a57f04985b4d713b7118d6b/internal/events/subscription.go#L363-L365